style.css

```diff
@@ -0,0 +1,689 @@
+
+/* global */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f1f1f1;
+  max-width: 100%;
+  filter: grayscale(1);
+}
+
+body section,
+body footer,
+body input {
+  font-family: 'Balsamiq Sans', cursive;
+}
+
+.flex {
+  display: flex;
+}
+
+.justify-content-space-around {
+  justify-content: space-around;
+}
+
+.align-items {
+  align-items: center;
+}
+
+.grid {
+  display: grid;
+}
+
+.black-hashtag {
+  color: #555;
+}
+
+.btn {
+  padding: 0.75rem 1.5rem;
+  background-color: #fff;
+  color: dodgerblue;
+  border: transparent;
+  border-radius: 7px;
+  font-weight: 700;
+}
+
+.arrow-icon {
+  position: absolute;
+  left: 80px;
+  bottom: 20px;
+  font-size: 32px;
+}
+
+.vertical {
+  position: absolute;
+  transform: rotate(90deg);
+  left: -30px;
+  top: 150px;
+}
+
+.margin-bottom-150 {
+  margin-bottom: 150px;
+}
+
+.see-more {
+  text-align: center;
+  padding: 0.5rem 0;
+  padding: 60px 0;
+}
+
+.see-more-link {
+  border-bottom: 3px solid #d33a2c;
+  text-decoration: none;
+}
+
+body h2 {
+  font-size: 2.8rem;
+  line-height: 2.8rem;
+}
+
+.rotate-on-hover {
+  transform: rotate(-15deg);
+  transition: ease-in 0.35s;
+  transform-origin: bottom left;
+}
+
+.clear-fix {
+  clear: both;
+  display: table;
+  content: "";
+}
+
+.date {
+  color: #333;
+  font-size: 0.8rem;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.fundamento {
+  font-family: fondamento, cursive;
+}
+
+.white-color {
+  color: white;
+}
+
+/* links */
+
+a {
+  color: inherit;
+}
+
+.bio a {
+  color: blue;
+}
+
+.footer-options a {
+  margin-right: 10px;
+}
+
+.top-four-articles-grid a {
+  color: white;
+  text-decoration: none;
+  font-weight: 700;
+}
+
+.community-links-article-title a {
+  text-decoration: none;
+}
+
+.footer ul li a {
+  text-decoration: none;
+}
```

Comment:
- [ ] You are repeating the same rules in different parts of your CSS. For good coding practices ([check this link](https://speckyboy.com/good-bad-css-practices)), follow the DRY rule (Don't repeat yourself).
![image](https://user-images.githubusercontent.com/53256139/89356769-33203f80-d684-11ea-8a2b-7776e5a73811.png)

Reply:
summarized text-decoration: none; in one rule